lib/travis/api/v3/cron.rb

```diff
@@ -0,0 +1,5 @@
+module Travis::API::V3
+  class Cron
+    SCHEDULER_INTERVAL = 1.minute
```

Comment:
Is there going to be other logic or methods on this class? I might suggest moving this constant to the model class.